swap/protocol_test.go

```diff
@@ -47,7 +47,7 @@ func TestHandshake(t *testing.T) {
 		t.Fatal(err)
 	}
 	// setup the protocolTester, which will allow protocol testing by sending messages
-	protocolTester := p2ptest.NewProtocolTester(swap.owner.privateKey, 2, swap.run)
```

Comment:
same here, as commented above (i will refrain from repeating it from here onwards)